DiamondLightSource/dodal
# Add 16 filter attenuator, with set transmission function

Fixes #82 

To test:
Check new unit test
Confirm that the set_transmission() provides the same functionality is the same function in GDA (in EPICSAttenuatorBase.java)
